Structure.Rmd

```diff
@@ -0,0 +1,398 @@
+```
+
+The main differences between a source package and an uncompressed bundle are:
+
+* Vignettes have been built, so rendered outputs, such as HTML, appear below
+  `inst/doc/` and a vignette index appears in the `build/` directory, usually
+  alongside a PDF package manual.
+
+* A local source package might contain temporary files used to save time during
+  development, like compilation artefacts in `src/`. These are never found in 
+  a bundle.
+
+* Any files listed in `.Rbuildignore` are not included in the bundle. These are
+  typically files that facilitate your development process, but that should be
+  excluded from the distributed product.
+
+#### `.Rbuildignore` {#rbuildignore}
+
+You won't need to contemplate the exact structure of package `.tar.gz` files very often, but you do need to understand the `.Rbuildignore` file. It controls which files from the source package make it into the downstream forms.
+
+Each line of `.Rbuildignore` is a Perl-compatible regular expression that is matched, without regard to case, against the path to each file in the source package, i.e. the result of `dir(full.names = TRUE, recursive = TRUE, include.dirs = TRUE, all.files = TRUE)` run from the package's top-level directory. If the regular expression matches, that file or directory is excluded. Note there are some default exclusions implemented by R itself, mostly relating to classic version control systems and editors, such as SVN, Git, and Emacs.
+
+To exclude a specific file or directory (the most common use case), you __MUST__ anchor the regular expression. For example, to exclude a directory called "notes", use `^notes$`. The regular expression `notes` will match any file name containing notes, e.g. `R/notes.R`, `man/important-notes.R`, `data/endnotes.Rdata`, etc. The safest way to exclude a specific file or directory is to use `usethis::use_build_ignore("notes")`, which does the escaping for you.
+
+`.Rbuildignore` is a way to resolve some of the tension between the practices that support your development process and CRAN's requirements for submission and distribution. The affected files fall into two broad, semi-overlapping classes:
+
+  * Files that help you generate package contents programmatically. Examples:
+    - Using `README.Rmd` to generate an informative and current `README.md`.
+    - Storing `.R` scripts to create and update internal or exported data.
+  * Files that drive package development, checking, and documentation, outside
+    of CRAN's purview. Examples:
+    - Files relating to the RStudio IDE.
+    - Using the [pkgdown package](https://pkgdown.r-lib.org) to generate a
+      website.
+    - Configuration files related to continuous integration/deployment and 
+      monitoring test coverage.
+
+Here is a non-exhaustive list of typical entries in the `.Rbuildignore` file for a package in the tidyverse:
+
+```
+^.*\.Rproj$         # Designates the directory as an RStudio Project
```

Comment:
I think you should say "Ignores the RStudio project file"

Reply:
I'm not doing this because then it seems very un-parallel to the comments describing all the other lines. Let's discuss if you feel strongly I'm wrong about this.